Resources/Prototypes/Nyanotrasen/Roles/Jobs/Epistemics/forensicmantis.yml

```diff
@@ -39,7 +38,7 @@
     gloves: ClothingHandsGlovesColorWhite
     outerClothing: ClothingOuterCoatMantis
     belt: ClothingBeltMantis
-    # pocket2: ForensicScanner # DeltaV - Mantis is no longer a Detective
```

Comment:
I wouldn't mind Mantis not being a Detective, but then what are they? :thinking: 
I'd need to know what DeltaV has actually done with them before I decide what we should do with them.

Reply:
Delta-V has changed nothing of the Mantis, maybe resprited some things, I don't remember.